e2e/transport.go

```diff
@@ -42,10 +42,41 @@ func (t *transport) Gossip(msg *pbft.MessageReq) error {
 }
 
 type transportHook interface {
-	Connects(from, to pbft.NodeID) bool
```

Comment:
Please revert this change. I am fine changing the naming of the interface but not as part of a PR that is addressing something else.